.github/workflows/pr-build.yaml

```diff
@@ -0,0 +1,365 @@
+      - name: Integration Tests
+        run: |
+          ./mvnw -V -B \
+            -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn clean verify \
+            -Dnative \
+            -Ddocker \
+            -DskipTests \
+            -Pno-apache-snapshots \
+            -pl :camel-quarkus-integration-test-aws \
+            -pl :camel-quarkus-integration-test-consul
+  misc:
+    runs-on: ubuntu-latest
+    needs: build
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Set Up Java
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+      - name: Restore Cache
+        uses: actions/cache@v1
+        with:
+          path: ~/.m2/repository
+          key: maven-${{ github.sha }}
+          restore-keys: |
+            maven-${{ github.sha }}
+      - name: Integration Tests
+        run: |
+          ./mvnw -V -B \
+            -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn clean verify \
+            -Dnative \
+            -Ddocker \
+            -DskipTests \
+            -Pno-apache-snapshots \
+            -pl :camel-quarkus-integration-test-bean-validator \
+            -pl :camel-quarkus-integration-test-dozer \
+            -pl :camel-quarkus-integration-test-ftp \
+            -pl :camel-quarkus-integration-test-infinispan \
+            -pl :camel-quarkus-integration-test-mail \
+            -pl :camel-quarkus-integration-test-pdf \
+            -pl :camel-quarkus-integration-test-salesforce \
+            -pl :camel-quarkus-integration-test-slack \
+            -pl :camel-quarkus-integration-test-tarfile \
+            -pl :camel-quarkus-integration-test-twitter \
+            -pl :camel-quarkus-integration-test-validator \
+            -pl :camel-quarkus-integration-test-zipfile
+  # memoryhogs:
+  #   runs-on: ubuntu-latest
+  #   needs: build
+  #   steps:
+  #     - name: Checkout
+  #       uses: actions/checkout@v2
+  #     - name: Set Up Java
+  #       uses: actions/setup-java@v1
+  #       with:
+  #         java-version: 1.8
+  #     - name: Restore Cache
+  #       uses: actions/cache@v1
+  #       with:
+  #         path: ~/.m2/repository
+  #         key: maven-${{ github.sha }}
+  #         restore-keys: |
+  #           maven-${{ github.sha }}
+  #     # run the :camel-quarkus-integration-test-fhir as standalone process
+  #     # as it building the native image including camel-fhir requires lot 
+  #     # of memory this we nee to limit it to avoid the process fail for OOM.
+  #     - name: Integration Tests
+  #       run: |
+  #         ./mvnw -V -B \
+  #           -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn clean verify \
+  #           -Dnative \
+  #           -Ddocker \
+  #           -DskipTests \
+  #           -Dnative-image.xmx=5g \
+  #           -pl :camel-quarkus-integration-test-fhir
```

Comment:
fhir does not pass at all on GH actions?

Reply:
The native image compilation fails for out of memory, I have not yet found any combination to make it working in a reasonable amount of time (with the default settings it fails after ~ 30 minutes)